### PR TITLE
Remove non existing ofcerc token and fix asset name for TERC

### PR DESCRIPTION
**Pull request by FloBitGo**
There is no `ERC` token. Therefore there can't be an `ofcerc` token and the underlying asset for `ofcterc` and `terc` should be `terc`, like all other ERC20 tokens.

Ticket: BG-28775

### CODE REVIEW

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -588,10 +588,9 @@ export const coins = CoinMap.fromCoins([
     '',
     'TST'
   ),
-  ofcerc20('ofcerc', 'ERC Token', 0, UnderlyingAsset.ERC),
   ofcerc20('ofcusdt', 'Tether', 6, UnderlyingAsset.USDT),
   ofcerc20('ofcusdc', 'USD Coin', 6, UnderlyingAsset.USDC),
-  tofcerc20('ofcterc', 'ERC Test Token', 0, UnderlyingAsset.ERC),
+  tofcerc20('ofcterc', 'Test ERC Token', 0, UnderlyingAsset.TERC),
   tofcerc20('ofctusdt', 'Test Tether', 6, UnderlyingAsset.USDT),
   tofcerc20('ofctusdc', 'Test USD Coin', 6, UnderlyingAsset.USDC),
```
Comment (jjn2009) [inline modules/statics/src/coins.ts:595]: as per our conversation in slack these basically dont exist I think? I've been directed to use TERC instead which suggests this actually doesnt exist on the ropstine network

 @marktoda

**Review by tylerlevine — DISMISSED** [review]

File: modules/statics/src/coins.ts (hunk 1 above)
Comment (marktoda) [reply modules/statics/src/coins.ts]: > ropstine network.


We use kovan tesnet not ropsten

**Review by marktoda — DISMISSED** [review]

File: modules/statics/src/coins.ts [hunk 2]
```diff
@@ -588,10 +591,4 @@ export const coins = CoinMap.fromCoins([
     '',
     'TST'
   ),
-  ofcerc20('ofcerc', 'ERC Token', 0, UnderlyingAsset.ERC),
-  ofcerc20('ofcusdt', 'Tether', 6, UnderlyingAsset.USDT),
-  ofcerc20('ofcusdc', 'USD Coin', 6, UnderlyingAsset.USDC),
```
Comment (jjn2009) [inline modules/statics/src/coins.ts:593]: these actually should exist in the future

Comment (jjn2009) [reply modules/statics/src/coins.ts:593]: just need to remove the testnet ones

**Comment by FloBitGo** [comment]
Sorry folks, I had to rewrite history to fix the commit message, I would appreciate another review.

**Review by jjn2009 — APPROVED** [review]

**Review by marktoda — APPROVED** [review]

**Pull request merged**